src/Forms/Checkbox.js

```diff
@@ -41,7 +41,7 @@ const Checkbox = React.forwardRef(({ checked, className, defaultChecked, disable
                 <input
                     {...inputProps}
                     aria-checked={getCheckStatus(checked, indeterminate)}
-                    checked={checked || defaultChecked}
+                    checked={typeof checked === 'undefined' ? defaultChecked : checked}
```

Comment:
Weird we don't have a test for indeterminate.

Reply:
Are you talking about `aria-checked={getCheckStatus(checked, indeterminate)}`?
```js
const isChecked = {typeof checked === 'undefined' ? defaultChecked : checked;
// ...
aria-checked={getCheckStatus(isChecked, indeterminate)}
```

Reply:
I added a test for indeterminate and others for `aria-checked`